FusionIIIT/applications/placement_cell/views.py

```diff
@@ -506,11 +502,10 @@ def delete_invitation_status(request):
         delete_invit_status_key = request.POST['deleteinvitationstatus']
 
         try:
-            PlacementStatus.objects.get(pk=delete_invit_status_key).delete()
+            PlacementStatus.objects.select_related('unique_id','notify_id').get(pk=delete_invit_status_key).delete()
             messages.success(request, 'Invitation Deleted Successfully')
         except Exception as e:
 
-            print('---- \n\n record not found')

```

Comment:
You deleted this print statement but didn't replace it with the logger. We may still need to catch these exceptions and display them somewhere.